go.mod

```
@@ -1,6 +1,6 @@
 module gopkg.in/DataDog/dd-trace-go.v1
 
-go 1.22.0
+go 1.22.10
```

```suggestion
go 1.22.0
```

We usually pin to the first version available.

---

Hmmm... I definitely did not intend this change... I wonder what made it sneak in!